### ISSUE TITLE
Microsoft Defender alarms

### ISSUE THREAD

**Issue by antaeusdk** [zen-browser/desktop#432]
### What happened?

When executing the zen.exe, 4 alarms go off with Microsoft Defender. It resulted in my computer getting isolated.
![zenbrowser](https://github.com/user-attachments/assets/cc458345-bfaf-4ccc-83ed-7001c5066b45)
The reason for this bug report, is just to let you know how much noise it makes in a cloud based corporate setup.

I can not provide logs, since I was ordered to remove all traces of the browser from my computer.
The installer was blocked by a function called Attack Surface Reduction, since the installer seemed to create child processes.
The portable opened just fine, but gave the alerts listed above.

Have a good day :)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

a0.26

### Severity impact

Medium

### What platform are you seeing the problem on?

Windows

**Comment by bdbch** [comment]
Are there plans to do the signing of keys soon? Probably a lot of potential users are currently put off of Defender systems (on all systems) throwing warnings because of missing publisher keys/information.

**Comment by Donnnno** [comment]
Appsigning is in progress! 
https://ko-fi.com/zen_browser/goal?g=210

**Comment by mauro-balades** [comment]
Duplicate of https://github.com/zen-browser/desktop/issues

**Comment by BloxDeGroteUnsklled** [comment]
I also recently had this issue. Even though there was no defender business I was informed of an overseas security breach, where my credentials could've been stolen. What is going on?
